Doc/conf.py

```diff
@@ -157,6 +157,77 @@
     ('envvar', 'USER'),
     ('envvar', 'USERNAME'),
     ('envvar', 'USERPROFILE'),
+]
+
+# Temporary undocumented names.
+# In future this list must be empty.
+nitpick_ignore = [
```

Comment:
Currently this overwites the list above:
```suggestion
```

A

Reply:
Oh, I was going to use `+=`.